src/integration-tests/tests/driver.rs

```diff
@@ -78,6 +78,16 @@ mod driver {
             .map_err(report)
     }
 
+    #[test_case(ta::client::TelcoAutomation::builder().with_tracing(); "with tracing enabled")]
+    #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
+    async fn run_telco_automation(
```

Comment:
Consider calling this `error_details`. That is what is under test. `telcoautomation` is an implementation detail.

Reply:
Done.